alldata/bblab_site/bblab_site/urls.py

```diff
@@ -23,6 +23,7 @@
 	path('', views.root_redirect),
 	path('account/', include('account.urls')),
 	path('tools/', include('tools.urls')),
+	path('django/wiki/', include('wiki.urls')),
 	path('wiki/', include('wiki.urls')),
```

Comment:
Should this line be removed, if the pages have moved to `django/wiki`?

Reply:
Yes, it can be removed :+1: